Lib/fontbakery/commands/check_ufo_sources.py

```diff
@@ -5,9 +5,9 @@
 import sys
 from functools import partial
 
-from fontbakery.commands.check_spec import main as super_main
-from fontbakery.commands.check_spec import \
-    runner_factory as super_runner_factory
+from fontbakery.commands.check_specification import main as super_main
+from fontbakery.commands.check_specification import (runner_factory as
+                                                     super_runner_factory)
```

Comment:
maybe these 2 lines could be merged since they import from the same module.

Reply:
Yeah :) That's the result of sorting imports in VS Code.